src/Anywhere.ArcGIS/Operation/ServiceLayerDescription.cs

```diff
@@ -191,4 +194,61 @@ public class AdvancedQueryCapabilities
         [DataMember(Name = "supportsQueryWithDistance")]
         public bool SupportsQueryWithDistance { get; set; }
     }
+
+    [DataContract]
+    public class LayerTimeInfo
+    {
+        [DataMember(Name = "startTimeField")]
+        public string startTimeField { get; set; }
```

Comment:
Can you make tall he properties camel case please e.g. `public string StartTimeField { get; set; }`

Reply:
Ah sorry, I haven't noticed that.. I used online generator to convert from json.